backend/tests/test_app.py

```diff
@@ -12,7 +12,7 @@ def response_get():
 
 def test_index_context(response_get):
     data = json.loads(response_get.data)
-    assert ap.msg_hello == data
+    assert data["greeting"] == "Hello World"
```

Comment:
If we will change json, we’ll have to change the code in the test

Reply:
I put the test for index page back as it has been written

Reply:
actually it's often better to hardcode data in the tests. 
So that the reader of the test can see all the information necessary information inside the test. 
In this case it doesn't matter much.

FYI for the future: https://testing.googleblog.com/2019/12/testing-on-toilet-tests-too-dry-make.html @DramaTourG @leopardilla